### ISSUE TITLE
Loop Coins: only one loop takes place

### ISSUE THREAD

**Issue by SigmaTau2003** [GeekMagicClock/smalltv-pro#47]
After switching on, all coins are looped once. Shouldn't this be a continuous loop?

Model: GeekMagic SmallTV-PRO,Version: V3.2.28EN

**Comment by GeekMagicClock** [comment]
Yes, should be continues loop, can you show me your coin settings?  But if the symbol is not correct, it will stop.
